### DISCUSSION TITLE
Giving up on Harvesting, it's just too hard. How do you import multiple files from one folder?

### DISCUSSION THREAD

**Discussion by BryanCrotaz** [wixtoolset/issues#7814] [Questions]
I have 227 files in my publish folder. I've been trying to use HarvestDirectory to bring them in but I've completely failed to get it to run on Github Actions, and can't see how to reference the exe in a ServiceInstall. So I'm giving up and doing it by hand (but not looking forward to typing out 227 components and files!)

It seems that you have to put something called a `BindPath` on a `File` element? Is this the correct way to set where the file is imported from? How do you import lots of files from one folder? Is there a ParentDirectory element? Component doesn't have a BindPath you see.

**Comment by robmen** [comment]
In [Episode 44 of the Deployment Dojo](https://youtube.com/live/OndebYtdb3Y), I start a 3-part series on the different ways you can use to find your files.

Also, in [Episode 52 of the Deployment Dojo](https://youtube.com/live/XcUmwegYClg), I did exactly what you described in your first paragraph with Heat. And an hour ago, [in Episode 53](https://youtube.com/live/kg6nEBycoFs) (video may need a few hours to finish encoding), I did the same as Episode 52 but with the [Advanced Harvesting tools from FireGiant](https://www.firegiant.com/docs/heatwave-build-tools/harvesting/) which are much easier to use than Heat.
